src/api/spec/models/attrib_spec.rb

```diff
@@ -9,6 +9,18 @@
     it { expect(attribute.fullname).to eq("#{attribute.namespace}:#{attribute.name}") }
   end
 
+  context "#container" do
```

Comment:
> What is the difference to test in line 24-43? That you don't assign the container but instantiate with the object?

In lines 12-22 it is tested the `container` method. Here it is not needed to do a container assignment (with `container=`). Factories are used instead.

In lines 24-43 it is tested the `container=` method. Here, after doing the assignment, it is also tested the result of the containter method, but not only. It is also tested the package and the project attributes.

Both tests try to test only its method.